docker-compose.yml

```diff
@@ -0,0 +1,75 @@
+version: "3.4"
+
+volumes:
+  db:
+
+networks:
+  app:
+
+services:
+  database:
+    image: mariadb
+    ports:
+      - "127.0.0.1:3306:3306"
+    volumes:
+      - db:/var/lib/mysql
+    environment:
+      MYSQL_ROOT_PASSWORD: Abc123456
+      MYSQL_USER: meethut
+      MYSQL_PASSWORD: Abc123456
+      MYSQL_DATABASE: meethut
+    networks:
+      - app
+
+  frontend:
+    build:
+      context: ./MeetHut.Backend/ClientApp
+      dockerfile: Dockerfile
+    networks:
+      - app
+    volumes:
+      - ./MeetHut.Backend/ClientApp:/app
+
+  backend:
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args:
+        - USER=${USER}
+        - DUID=${DUID}
+        - DGID=${DGID}
```

Comment:
I also had to specify the platform of the backend service, otherwise it would use the default `linux/arm64` platform of mine and would fail finding the image.

So it either should be specified at the service params:

```yaml
backend:
    platform: linux/amd64
    build:
      context: .
      dockerfile: Dockerfile
      args:
        - USER=${USER}
        - DUID=${DUID}
        - DGID=${DGID}
```


or parameterised:

```yaml
backend:
    platform: ${PLATFORM}
    build:
      context: .
      dockerfile: Dockerfile
      args:
        - USER=${USER}
        - DUID=${DUID}
        - DGID=${DGID}
        - PLATFORM=${PLATFORM}
```

Makefile:
```makefile
up:			## Start docker dev env
	DUID=$(shell id -u) DGID=$(shell id -g) PLATFORM="linux/amd64" docker-compose up
```